lib/streamlit/elements/file_uploader.py

```diff
@@ -41,6 +48,70 @@
 
 
 class FileUploaderMixin:
+    @overload
```

Comment:
The repetition in the parameters, types, and default arguments makes it hard to see what differs between them. Can you add a brief comment to each signature overload indicating what is different about it from the others? Eg `# accept_multiple_files = True`